docs/handbook/community.md

```diff
@@ -0,0 +1,42 @@
+<!--
+
+The contents of this Documentation are subject to the Public Documentation License Version 1.01
+ (the "License"); you may only use this Documentation if you comply with the terms of this License.
+A copy of the License is available at http://illumos.org/license/PDL.
+
+
+The Original Documentation is _________________.
+
+The Initial Writer of the Original Documentation is ___________ Copyright (C)_________[Insert year(s)].
+All Rights Reserved. (Initial Writer contact(s):________________[Insert hyperlink/alias]).
+
+Contributor(s): ______________________________________.
+
+Portions created by ______ are Copyright (C)_________[Insert year(s)].
+All Rights Reserved. (Contributor contact(s):________________[Insert hyperlink/alias]).
+
+-->
+
+<img src = "../../Openindiana.png">
+
+# Hipster Handbook - Community Tutorials
+
+
+## How to Install Oracle Database 11R2 on OpenIndiana Hipster
+
+Contributed by OpenIndiana community member Franklin Ronald, this article details the steps required for the installation of Oracle Database 11R2 on OpenIndiana Hipster.
+
+Link to PDF Document: <a href= "../pdf/HowToInstallOracleDB.pdf" target="_blank">How to Install Oracle Database 11R2 on OpenIndiana Hipster</a>
+
+<i class="fa fa-info-circle fa-lg" aria-hidden="true"></i> **NOTE:**
+<div class="well">
+<p>It is not recommended to run Oracle Database in OpenIndiana for production environments. Oracle has a restricted list of supported operating systems and unfortunately OpenIndiana is not in it. The purpose of this article is to install the Oracle Database for development environment.</p>
```

Comment:
s/in OpenIndiana for production environments/on OpenIndiana in production environment/.